src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationFactory.cs

```diff
@@ -161,6 +164,17 @@ private void SetContentRoot(IWebHostBuilder builder)
             }
         }
 
+        private bool TrySetContentRootUsingEnvironmentVariables(IWebHostBuilder builder)
+        {
+            var expectedVariableName = "AspNetCore.Mvc.WebRoot-" + typeof(TEntryPoint).Assembly.GetName().Name;
```

Comment:
I think we generally use `ALL_CAPS_WITH_UNDERSCORES` for environment variables.

I think this is for portability reasons, someone like @pranavkm or @natemcmaster  would be able to truly answer 😆  - so this means that we might need to mangle the assembly name as well which would likely contain `.` characters.

As for the environment variable name, I think something like `ASPNETCORE_TEST_CONTENTROOT_ASSEMBLY_NAME` would be more appropriate and consistent. For example 'web root' is it's own setting, so I don't want to overload that name.

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/web-host?view=aspnetcore-2.1&tabs=aspnetcore2x#content-root
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/web-host?view=aspnetcore-2.1&tabs=aspnetcore2x#web-root

Reply:
also FYI @Tratcher in case you have thoughts